src/Uno.UWP/Extensions/CoreDispatcherExtensions.cs

```diff
@@ -20,7 +20,7 @@ internal static async Task<T> RunWithResultAsync<T>(this CoreDispatcher dispatch
 		{
 			var tcs = new TaskCompletionSource<T>();
 
-			dispatcher.RunAsync(
+			await dispatcher.RunAsync(
```

Comment:
@jeromelaban Should this assign to a discard? I don't know if it should be fire-and-forget or not.

Reply:
Let's keep it that way. It's not used much, and not in critical paths so we can validate with the additional await.